Source/Engine/Level/SceneObjectsFactory.cpp

```diff
@@ -238,6 +239,8 @@ void SceneObjectsFactory::HandleObjectDeserializationError(const ISerializable::
     {
         const Guid parentId = JsonTools::GetGuid(parentIdMember->value);
         Actor* parent = Scripting::FindObject<Actor>(parentId);
+        DummyScript* dummyScript = parent->AddScript<DummyScript>();
+        dummyScript->Data = String(buffer.GetString());
```

Comment:
`MissingTypeName` could be cached here from the `const ISerializable::DeserializeStream& value`.

Reply:
Done + it cleaned up some code in MissingScript